demos/graphite/README.md

```diff
@@ -0,0 +1,12 @@
+# configure mesos plugin

```

Comment:
mesos?

Reply:
sorry my mistake, a quick and dirty copy/paste ... It is fix now